app/scripts/controllers/push-platform-notifications/push-platform-notifications.test.ts

```diff
@@ -13,6 +13,8 @@ const MOCK_FCM_TOKEN = 'mockFcmToken';
 const MOCK_TRIGGERS = ['uuid1', 'uuid2'];
 
 describe('PushPlatformNotificationsController', () => {
+  process.env.ENABLE_MV3 = 'true';
+
```

Comment:
```suggestion
const describeOnlyMV3 = process.env.ENABLE_MV3 ? describe : describe.skip;

describeOnlyMV3('PushPlatformNotificationsController', () => {
```

Instead of silently forcing MV3, which could cause confusion, the tests can be skipped on non-MV3-environments.

Reply:
Oooh, good callout!

Sure we can do this!

Reply:
@Prithpal-Sooriya Just following up here, why was it decided to mark the tests as "passing" on MV2 rather than skipping them?

https://github.com/MetaMask/metamask-extension/pull/24662/files#diff-4a248190569b3e87b4ee8d39ead794d3b56cfe181216f844040a4aafa3e1fad1R18